### PR TITLE
docs: Updated deployment/summary

### PR DESCRIPTION
**Pull request by crabba**
Issue #, if available:

Description of changes: docs/deployment/summary updated.

Checklist: 

Docs only

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by SanketD92 — APPROVED** [review]
LGTM

File: docs/docs/deployment/pre_deployment/configuration.md [hunk 1]
```diff
@@ -4,92 +4,64 @@ title: Configuration
...
 
-In this optional step you will make a copy of a sample global config
-file, name it for your stage, and modify it. Current default values for
-the main configuration are stored in
-`main/config/settings/.defaults.yml`. If no stage-named settings file is
-present, values will be read from this default file.
+The `settings` directory contains a file `.defaults.yml`, which is read when Service Workbench is deployed.  This file contains the settings for a default installation of Service Workbench.  If, in addition to the default configuration file, there is a file named after the stage name (`<stage>.yml`), this file is read and processed after the default file.  This stage-named file can be used to override settings established in the default configuration file.  A sample stage configuration file `example.yml` is provided as a template; copy or rename this file to your stage name for it to take effect.  You may find some configuration settings are present in the default file and not in the example file, so read these files carefully.
```
Comment (Bingjiling) [inline docs/docs/deployment/pre_deployment/configuration.md:21]: Maybe we can specify the minimal required config values here?

File: docs/docs/deployment/pre_deployment/deployment_instance.md [hunk 2]
```diff
@@ -3,35 +3,13 @@ id: deployment_instance
...
-```{=html}
-<!-- -->
-```
--   **node** and all packages are installed in eg:
-    `~/.nvm/versions/node/v12.18.1/bin/node`
--   Verify with eg: `hygen --version`
+* Use a T2.medium (4 GB) EC2 instance or larger
+    * Larger machines will have faster networking; larger disks will have higher performance
+    * Default VPC & subnet are sufficient
+* You can avoid using Profiles by attaching to your instance a IAM role with sufficient permission (such as AdministratorAccess)
```
Comment (Bingjiling) [inline docs/docs/deployment/pre_deployment/deployment_instance.md:15]: Minor - an IAM role

File: docs/docs/deployment/pre_deployment/prerequisites.md [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
-------------
-Service Workbench can be deployed to an existing AWS Organization master account, or a member account, or some other account not part of the AWS Organization.
+
+Service Workbench can be deployed from either a master or member account of an AWS Organization, or from an account not associated with an Organization.  The choice is determined by customer preference.  The account in which Service Workbench is deployed, is known as the Service Workbench *Main* account.  During the installation you will require access to this main account, either through direct login, or through an AWS Command Line Interface [Profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)
+
+## Billing Account
+
+Within Service Workbench, all resources are deployed against an account which is responsible for charges incurred by that resource.  Billing accounts can be either **Created** by Service Workbench (vending an account within the Organization), or can be **Added** (importing an existing external account for billing purposes).  There can be multiple billing accounts set up within Service Workbench, with projects created under each account.
```
Comment (Bingjiling) [inline docs/docs/deployment/pre_deployment/prerequisites.md:13]: Is 'Billing Account' the same as 'Member Account'?

Comment (crabba) [reply docs/docs/deployment/pre_deployment/prerequisites.md:13]: I use Master/Member only in the context of an Organization.  I use the term 'Billing account' for an Account that has been added to SWB through the 'Add Account' mechanism.  The term 'Account' is used 5 different ways in SWB...so I used the term 'Billing account' here.  Perhaps saying 'These billing accounts can be...' makes it clearer that it refers to Added or Created accounts.

**Pull request merged**